api/tests/unit/domain/models/CertificationIssueReportResolutionStrategies_test.js

```diff
@@ -549,22 +345,23 @@ describe('Unit | Domain | Models | CertificationIssueReportResolutionStrategies'
       });
     });
 
-    context('the challenge does not contain an embed', function () {
+    context('the challenge does not contain an image or an embed', function () {
```

Comment:
Ca passe, mais limite...
```suggestion
    context('the challenge does not contain an image nor an embed', function () {
```

Reply:
Done.